src/App.tsx

```diff
@@ -76,15 +73,15 @@ export const App: React.FC = () => {
                   </div>
                 )}
 
-                {author && loaded && !hasError && posts.length === 0 && (
+                {author && loaded && !hasError && items.length === 0 && (
```

Comment:
Use logical operators 
```suggestion
                {author && loaded && !hasError && !items.length && (
```